### ISSUE TITLE
Some minor bugs with the latest update

### ISSUE THREAD

**Issue by hazemaze** [raiguard/Factorio-EditorExtensions#83]
#### Describe the Bug

After the last update, I noticed a few things when I tested the map editor. I don't know if it's a bug, but it didn't feel right.

I'm currently playing K2. When I started the map editor, I unlocked the achievement atomic bomb, which is now also displayed in my current save. I'm not that experienced with K2, but if I'm looking at this right, I don't have an atomic bomb yet.

I made a rocket silo to test something. After leaving the map editor, the item got transferred to my normal world. I then tried to remove it again. Put it in a box first and tried to shoot it, which didn't work and is probably normal for K2. However, I couldn't remove/destroy it in the map editor either. I put it in an infinity chest and enabled remove unfiltered items, but the item wasn't removed. The infinity chest was created with force EE_Testforce_1, if that's helpful.

Finally, I loaded an autosave to get rid of the rocket silo.

#### To Reproduce

1. Start map editor
2. Craft an item
3. Leave map editor
4. Item is also available in normal world.

Be certain to reproduce the issue on the same save file that you upload.

#### Save file & Username

[Save file](https://www.dropbox.com/s/mxor5rvf8d1gy5i/1.1.50%20Krastorio%202%202nd%20run%20with%20alien%20biomes.zip?dl=0)
Username: Lmp

#### Log file

[factorio-current.log](https://github.com/raiguard/Factorio-EditorExtensions/files/7841909/factorio-current.log)

**Comment by raiguard** [comment]
The mod syncs your inventory between the editor and regular play by default. You can disable that in the per player settings.

**Comment by hazemaze** [comment]
I've overlooked this setting and the tooltip as well. Thanks for the hint.

I'm still confused about the achievement. I was referring about the wrong achievement. I don't mean the atomic bomb, but antimatter bomb (destroyer of worlds).
As far as I can tell, I haven't researched those yet. A few days ago, I also got the achievement digging deep, but I don't remember placing a quarry drill anywhere. I don't even have one in my inventory. However, this was already before the new lab functionality was available in the map editor.
